TECHNICAL-SETUP.md

```diff
@@ -0,0 +1,67 @@
+# Technical setup 
+
+
+## Software requirement
+
+To build the project:
+
+- [Golang](https://golang.org/dl/) v1.16 or greater 
+- [`make`](https://www.gnu.org/software/make/) - to use Makefile targets
+- `sha256sum` = to calculate build checksums
+
+To run the [seed scripts](scripts/seeds/README.md)
+
+- OS: linux or macOS
+- [jq](https://stedolan.github.io/jq/) - json processor
+
+For git pre-commit hooks
+
+- python
+- [pre-commit](https://pre-commit.com/#install) - git pre-commit hooks framework 
+- [golangci-lint](https://github.com/golangci/golangci-lint) - linter for Go 
+
+### Swagger 
+
+To generate the swagger UI the following dependencies must be installed
+
+- [`swagger-combine`](https://www.npmjs.com/package/swagger-combine)
+
+
+## Development environment setup (only for contributing)
+
+For development it is highly recommended to install the configured pre-commit hooks by running from the project's root folder:
+
+```sh
+$ pre-commit install
+$ pre-commit install --hook-type commit-msg
+```
+
+This operations should be run only once. 
```

Comment:
````suggestion
Install the configured pre-commit hooks by running these commands from the project's root folder:

```sh
$ pre-commit install
$ pre-commit install --hook-type commit-msg
```
````

Reply:
I wonder if we need line 39 here? I deleted

> This operations should be run only once. 

Probably not, since no other commands or setups specify to run commands or install anything more than once. Is there a risk if the pre-commit install  commands are run more than once?